src/rust/src/wallet.rs

```diff
@@ -1395,3 +1395,13 @@ pub extern "C" fn orchard_wallet_init_from_frontier(
         false
     }
 }
+
+#[no_mangle]
+pub extern "C" fn orchard_wallet_commitment_tree_complete(wallet: *const Wallet) -> bool {
```

Comment:
"Commitment tree complete" invokes a very different idea in my head (i.e. that the global commitment tree is full and the shielded pool cannot be used anymore).
```suggestion
pub extern "C" fn orchard_wallet_unspent_notes_are_spendable(wallet: *const Wallet) -> bool {
```